src/components/timer/timer.tsx

```diff
@@ -93,7 +93,7 @@ const Timer = (props: Props) => {
           <>
             <div className={styles.timerColon}>:</div>
             <div className={styles.timerWrap}>
-              <div className={styles.timerNumber}>{timerSeconds}</div>
+              <div className={`visual-test:invisible ${styles.timerNumber}`}>{timerSeconds}</div>
```

Comment:
Why do we want to hide this?

Reply:
The seconds counter is something that will probably cause snapshots to differ over runs, as a small delay in the test could cause the second to tick over and cause a visual diff